sdcm/sct_config.py

```diff
@@ -172,7 +172,17 @@ class SCTConfiguration(dict):
                   Test duration (min). Parameter used to keep instances produced by tests
                   and for jenkins pipeline timeout and TimoutThread.
              """),
-
+        dict(name="prepare_duration", env="SCT_PREPARE_DURATION", type=int,
```

Comment:
Can we name it `stress_prepare_duration`?
It sounds less ambiguous to me.
Also, it becomes more familiar with the second new option - `stress_duration`.

Reply:
@aleksbykov you said yes, but you didn't change it..

Reply:
done